### PR TITLE
Fix login

### PR DESCRIPTION
**Pull request by Jonas-WMDE**

### CODE REVIEW

File: tests/browser/environments.yml [hunk 1]
```diff
@@ -15,9 +15,9 @@
 #   bundle exec cucumber
 #
 mw-vagrant-host: &default
-  mediawiki_url: http://127.0.0.1:8080/wiki/
-  mediawiki_user: Selenium_user
-  mediawiki_password: vagrant
+  mediawiki_url: http://localhost/mediawiki/index.php/
+  mediawiki_user: admin
+  mediawiki_password: wikiwiki
```
Comment (thiemowmde) [inline tests/browser/environments.yml]: Is this mistakenly submitted from your local environment? Neither the commit message nor the pull request summary explain this.

File: tests/browser/environments.yml [hunk 2]
```diff
@@ -35,3 +35,4 @@ test2:
   # mediawiki_password: SET THIS IN THE ENVIRONMENT!
 
 default: *default
+
```
Comment (thiemowmde) [inline tests/browser/environments.yml]: Can you remove this please, for the most minimal diff?

File: tests/browser/environments.yml (hunk 1 above)
Comment (JeroenDeDauw) [inline tests/browser/environments.yml]: You'd almost think the author of this PR would go outside for lunch, leaving his keys on his desk. Oh wait...

/me hides under a table

Comment (thiemowmde) [inline tests/browser/environments.yml]: LOL.

File: tests/browser/features/support/pages/login_page.rb [hunk 3]
```diff
@@ -13,11 +13,13 @@ class LoginPage
   text_field(:password, id: 'wpPassword1')
   button(:login, id: 'wpLoginAttempt')
   link(:logout, xpath: "//li[@id='pt-logout']/a")
+  li(:userpage, id: 'pt-userpage')
```
Comment (thiemowmde) [inline tests/browser/features/support/pages/login_page.rb]: Please note that this does not really check for a successful login. This checks if the link to the user page is present in the top right corner of the interface, indicating that *some* user is logged in.

Good enough for what it's supposed to do, just wanted to make this clear.

File: tests/browser/features/support/pages/delete_item_page.rb [hunk 4]
```diff
@@ -10,9 +10,11 @@ class DeleteItemPage
   include PageObject
 
   button(:delete, id: 'wpConfirmB')
+  p(:return_to, id: 'mw-returnto')
```
Comment (thiemowmde) [inline tests/browser/features/support/pages/delete_item_page.rb]: Please note that this element appears after different actions.

Good enough for this patch, just wanted to mention that.

File: tests/browser/environments.yml [hunk 5]
```diff
@@ -18,7 +18,7 @@ mw-vagrant-host: &default
   mediawiki_url: http://127.0.0.1:8080/wiki/
   mediawiki_user: Selenium_user
   mediawiki_password: vagrant
-
+
```
Comment (thiemowmde) [inline tests/browser/environments.yml]: Can you remove this please?

**Pull request merged**